example-code/app/src/backend/test/Bridge.test.ts

```diff
@@ -163,7 +172,7 @@ describe.skip("Bridge", async () => {
   before(async () => {
     await IModelHost.startup();
     requestContext = await TestUtility.getAuthorizedClientRequestContext(TestUsers.superManager);
-    testProjectId = (await queryProjectIdByName(requestContext, "iModelJsIntegrationTest")).wsgId;
+    testProjectId = (await getITwinByName(requestContext, "iModelJsIntegrationTest")).id;

```

Comment:
Can you just remove this whole test? It's skipped and would mean you don't even need the queryITwin in here.

Reply:
Do we know why it is skipped? This entire file is well commented and detailed, but it boils down to only one test that is being skipped which is very strange.